packages/explorer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.1",
+  "private": true,
```

Comment:
either we should remove this or, if we're not yet ready to publish, remove the changeset

Reply:
the idea is to low-key publish it already, removed `private` 😬